tests/test_public_cli.py

```diff
@@ -179,7 +194,7 @@ def test_partial_run(
         config,
         cmdline="pyani fastani ...",
         fasta_directory=input_genomes_tiny,
-        status="Partial",
+        status="Running",  # simulate a partial run not ended cleanly
```

Comment:
Changing to "Running" to hit a branch in the delete-run code which flags that with an extra message and confirmation unless using ``--force``.